Testing/Unit Tests.md

```diff
@@ -85,18 +65,22 @@ And that's it, with this simple change we can make this method think it is any t
 
 ``` c#
 [Theory]
-[InlineData(6, 0)]
-[InlineData(10, 0)]
-[InlineData(11, 59)]
-public void GetTimeBasedGreeting_WhenItIsMorning_ThenReturnGoodMorning(
+[InlineData(11, 59, "Good morning")]
+[InlineData(12, 0, "Good afternoon")]
+[InlineData(18, 01, "Good evening")]
+public void GetTimeBasedGreeting_WhenTimeAvailable_ThenReturnAppropriateGreeting(
     int hour, 
-    int minutes)
+    int minutes,
+    string expectedGreeting)
 {
-    _timeProviderMock.Setup(m => m.UtcNow)
+    _timeProvider
+        .GetUtcNow()
         .Returns(new DateTime(2042, 2, 4, hour, minutes, 0));
 
     var result = _sut.GetTimeBasedGreeting();
 
-    result.Should().BeEquivalentTo("Good morning");
+    result.Should().BeEquivalentTo(expectedGreeting);
 }
-```
\ No newline at end of file
+```
+
+TimerProvider class enables us to do much more than just work with current time, so for more examples check out this [article](https://andrewlock.net/exploring-the-dotnet-8-preview-avoiding-flaky-tests-with-timeprovider-and-itimer/).
```

Comment:
Here I recommend replacing "enables us to do much more" with "makes it possible to do much more". Since this is a technical article, I think it's better to be less personal. :)